src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.AddCommand;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+import duke.command.Command;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-dd-MM HHmm");
+
+    @Test
+    public void parse_byeCommand_returnsExitCommand() throws DukeException {
```

Comment:
I like how you name your test methods, it is very clear to me what you are testing for and what is the expected result.